### DISCUSSION TITLE
Custom keybindings

### DISCUSSION THREAD

**Discussion by KevinGimbel** [vcync/modV#516] [Ideas]
I'd like to define custom keybindings for various actions, such as toggling groups active or inactive, "tabbing" through groups,  quickly enable / disable a certain mod, ... I guess there are a lot of possibilities and use cases.

**Comment by 2xAA** [comment]
Hey Kevin!

This sounds like a great idea.
Do you mind if we transfer this discussion to an issue?

**Reply by KevinGimbel** [reply]
not at all, feel free to transfer!

**Reply by 2xAA** [reply]
Turns out you can't transfer Discussion into issues, only the other way around.
I'll create an issue and reference this discussion.

**Reply by 2xAA** [reply]
https://github.com/vcync/modV/issues/517
